services/monitoring/grafana/terraform/.gitignore

```diff
@@ -0,0 +1,6 @@
+main.tf
+.terraform/** */
```

Comment:
Q: this `*/` is to ignore all files but it does not embrace hidden files, correct?

Reply:
I think this is just a mistake

Reply:
thanks for highlighting